_data/ankita1329/ankita1329.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Ankita Singh # No longer than 28 characters
+institution: Galgotias university 🚩 # no longer than 58 characters
+profile_pic: ankita.jpeg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.

```

Comment:
Hey @ankita1329 ,please make it exact as per the image file ( with extension). So `jpeg` -> `jpg`

Reply:
I've changed that also